tests/sentry/incidents/test_subscription_processor.py

```diff
@@ -79,6 +79,15 @@
 
 @freeze_time()
 class ProcessUpdateBaseClass(TestCase, SpanTestCase, SnubaTestCase):
+    kwargs_keys = [
+        "action",
+        "incident",
+        "project",
+        "new_status",
+        "metric_value",
+        "notification_uuid",
+    ]
```

Comment:
Please derive this from the function itself. Having this args list manually defined is more fragile.